src/Microsoft.ServiceFabric.Actors/Runtime/ActorBase.cs

```diff
@@ -138,14 +138,14 @@ protected virtual Task OnDeactivateAsync()
         }
 
         /// <summary>
-        /// This method is invoked by actor runtime just before invoking an actor method. Override this method
-        /// for performing any actions prior to an actor method is invoked.
+        /// Overrides this method for performing any actions prior to an actor method is invoked.
```

Comment:
@v-rboado  Should be "Override".  A developer would need to override this method.